lib/openstax/aws/packer_1_4_1.rb

```diff
@@ -54,16 +54,12 @@ def run
       @logger.info("Running: #{command}")
 
       if !@dry_run
-        if @verbose && !@very_verbose
```

Comment:
Does `--verbose` serve any purpose anymore?

Also I think the two separate packer classes were made to keep the behavior consistent across versions so it might need a similar update.

Reply:
probably, Tom doesn't want it gone. 

@TomWoodward is the other packer class used? Do I apply the update there as well?

Reply:
I don’t think we are still using the old one.

Reply:
I don’t think we are still using the old one.

Reply:
Should I remove this old packer version? @jpslav

Reply:
i initially thought --version would now be useless and we could rip it out, but i noticed that its actually setting a log level above so i told karina it was fine to keep

Reply:
@KarinaMendez2 can you put the verbosity checks back the way they were?  I think it was working as intended.

1. If `@verbose` and `@very_verbose` are false, then the command is run without any logging output
2. If `@verbose` is true and `@very_verbose` is false, the command is run with packer logging on (`PACKER_LOG=1`) and filtering out everything except for `ui:` or `amazon-ebs:` lines.
3. If `@verbose` is true and `@very_verbose` is true, the command is run with packer logging on and nothing is filtered out (so you get it all).

Ditto for the other packer file too.

Reply:
> If @verbose and @very_verbose are false, then the command is run without any logging output

this is the behavior that i didn't want and asked to remove, there is no use case for running this without the output